ssc/cmod_cashloan.cpp

```diff
@@ -91,30 +91,29 @@ static var_info vtab_cashloan[] = {
     { SSC_OUTPUT, SSC_NUMBER, "total_cost", "Total installed cost", "$", "", "Financial Metrics", "*", "", "" },
 		
 
-	{ SSC_OUTPUT,        SSC_ARRAY,      "cf_energy_net",      "Energy",                  "kWh",            "",                      "Cash Flow",      "*",                     "LENGTH_EQUAL=cf_length",                "" },
-	{ SSC_OUTPUT,        SSC_ARRAY,      "cf_energy_value",      "Value of electricity savings",                  "$",            "",                      "Cash Flow",      "*",                     "LENGTH_EQUAL=cf_length",                "" },
+    { SSC_OUTPUT,       SSC_ARRAY,      "cf_energy_net",            "Electricity to grid net",       "kWh", "", "Cash Flow Electricity", "*", "LENGTH_EQUAL=cf_length", "" },
```

Comment:
Another potential confusing configuration is the stand alone battery / residential, which has monthly outputs 
<img width="1493" alt="image" src="https://user-images.githubusercontent.com/6498311/153393051-4d7f4175-eacc-43f9-9316-53d2456e2370.png">

The cash flow for the "Electricity to grid net (kWh)" term, 1814kWh, shows the "Energy to load from battery (kWh)" monthly sum for year 1 (potentially a sign issue compared to no battery?):
<img width="1316" alt="image" src="https://user-images.githubusercontent.com/6498311/153393539-774afe3e-3705-4e73-92c5-3d73f7697707.png">